dev/com.ibm.ws.webcontainer.servlet.3.1_fat.2/test-applications/LibertyWriteListenerFilterTest.war/resources/WEB-INF/web.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2014 IBM Corporation and others.
```

Comment:
CC?

Reply:
This file was just copied over from the com.ibm.ws.webcontainer.servlet.3.1_fat bucket: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.webcontainer.servlet.3.1_fat/test-applications/LibertyWriteListenerFilterTest.war/resources/WEB-INF/web.xml

I think git had some issues trying to figure out what files were copied and what were not. Since it was just an existing file that moved locations I didn't update the copyright header at all.